Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -203,19 +203,15 @@ open class AVFoundationPlayback: Playback {
 
     open override func play() {
         guard canPlay else { return }
-
-        if player == nil {
-            setupPlayer()
-        }
-
+        setupPlayerIfNeeded()
```

Comment:
It's a good point in a better way, but I think we are adding more than one responsibility to this method.

Reply:
the more than one responsibility was already there. The goal of this pull request is to reduce the `cognitive complexity` around it.